torch/fx/graph_module.py

```diff
@@ -825,10 +864,49 @@ def recompile(self) -> PythonCode:
         python_code = self._graph.python_code(root_module="self")
         self._code = python_code.src
         self._lineno_map = python_code._lineno_map
+        self._prologue_start = python_code._prologue_start
 
         cls = type(self)
         co_fields = self._graph._co_fields if hasattr(self._graph, "_co_fields") else {}
-        cls.forward = _forward_from_src(self._code, python_code.globals, co_fields)
+
+        if co_fields:
+            cls.forward = _forward_from_src(self._code, python_code.globals, co_fields)
+        else:
+            node_metadata: dict[int, dict[str, Any]] = {}
+            for i, node in enumerate(self._graph.nodes):
+                node_metadata[i] = {
```

Comment:
Question: Do we want to guard this part of the code behind a flag or make it the default behavior?

Reply:
The main problem with making this default behavior is you're going to affect ALL of the FX codegen happening everywhere in the company, and so if there are any bugs you are getting yanked. The second problem is that this code might actually be temporary if we go from aot eager to inductor, inductor takes care of making the files anyway, so we wouldn't need this code. So I definitely vote flag. It's also not obvious to me that we are adequately flagged right now, what exactly is `co_field`?